src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -889,42 +889,6 @@ void LottieBuilder::updateImage(LottieGroup* layer)
 }
 
 
-static float _pathLength(const PathCommand* cmds, uint32_t cmdsCnt, const Point* pts, uint32_t ptsCnt)
```

Comment:
@mgrudzinska This unnecessary addition/removal is not good. Please first create a path length refactoring patch in this PR.

Reply:
@mgrudzinska please check this: https://github.com/thorvg/thorvg/pull/3276